### PR TITLE
Changes that will fix a couple of remaining open issues left from the previous PR

### PR DESCRIPTION
**Pull request by PerAsperaAdAstra3**
- In all places like this where you have get http method and receive ID as an input parameter, you cannot be sure that thst string will be a number (force browsing). So, parseLong could raise an exception. Validation is necessary. Same with post method, but the risk of invalid data is smaller - DONE

- I think it's better if the conversion takes place on the server. But you can always convert values in the constructors, right? - ** I think its DONE **

- I think that statuses should also have constraints in the database - update the models. Let's change everything that requires changes to models now, including the booleans. But do not implement validations in this PR - DONE

- I think that statuses should also have constraints in the database - update the models. Let's change everything that requires changes to models now, including the booleans. But do not implement validations in this PR - DONE

- Think about where cascade all makes sense and where it does not. When is it better to just not allow deletion - DONE

- Very minor thing, but these erlxveption messages are not consistent across these service classes. Maybe create a custom exception and define a part of the message and provide the class bane as the input parameter)

- I don't think it's a great idea to insert a generic admin due to security concerns. You have to remember to delete it later. I'm fine with merging this now as long as we remove it later

- The colors could be defined as constants elsewhere. That would make it easier to adjust them if necessary

### CODE REVIEW

File: src/main/java/training/controller/RestClientManagementController.java [hunk 1]
```diff
@@ -125,8 +125,8 @@
 		obj.put("clientPackageId", clientPackage.getId());
 		obj.put("clientPackagePayed", clientPackage.isPayed());
 		
-		obj.put("clientPackageActive", clientPackage.isClientPackageActive());
-		if(clientPackage.isClientPackageActive()) {
+		obj.put("clientPackageActive", clientPackage.getClientPackageActive()); //clientPackage.isClientPackageActive());
```
Comment (renatav) [inline src/main/java/training/controller/RestClientManagementController.java:128]: remove comment

File: src/main/java/training/controller/RestClientManagementController.java [hunk 2]
```diff
@@ -125,8 +125,8 @@
...
+		if(clientPackage.getClientPackageActive() == ClientPackageStateEnum.ACTIVE) { //isClientPackageActive()) {
```
Comment (renatav) [inline src/main/java/training/controller/RestClientManagementController.java:129]: remove comment

File: src/main/java/training/converter/ClientPackageDTOtoClientPackage.java [hunk 3]
```diff
@@ -36,12 +36,11 @@ public ClientPackage convert(ClientPackageDTO source) {
 		clientPackage.setPayed(source.getPayed());
 		
 		if(source.getClientPackageActive().equals(ClientPackageStateEnum.ACTIVE.getNameText())) {
-			clientPackage.setClientPackageActive(true);
+			clientPackage.setClientPackageActive(ClientPackageStateEnum.ACTIVE); //true);
```
Comment (renatav) [inline src/main/java/training/converter/ClientPackageDTOtoClientPackage.java:39]: remove comment

File: src/main/java/training/converter/ClientPackageDTOtoClientPackage.java [hunk 4]
```diff
@@ -36,12 +36,11 @@ public ClientPackage convert(ClientPackageDTO source) {
...
 		} else {
-			clientPackage.setClientPackageActive(false);
+			clientPackage.setClientPackageActive(ClientPackageStateEnum.NOTACTIVE); //false);
```
Comment (renatav) [inline src/main/java/training/converter/ClientPackageDTOtoClientPackage.java:41]: remove comment

File: src/main/java/training/model/ClientPackage.java [hunk 5]
```diff
@@ -26,7 +28,8 @@
 	private Package packageUnitCP;
 	
 	@Column(name = "clientPackageStatus")
-	private boolean clientPackageActive;
+//	private boolean clientPackageActive;
```
Comment (renatav) [inline src/main/java/training/model/ClientPackage.java:31]: Remove

File: src/main/java/training/model/ClientPackage.java [hunk 6]
```diff
@@ -57,18 +60,26 @@ public void setClientPackagePrice(Long clientPackagePrice) {
 		this.clientPackagePrice = clientPackagePrice;
 	}
 
-	public boolean isClientPackageActive() {
+/*	public boolean isClientPackageActive() {
```
Comment (renatav) [inline src/main/java/training/model/ClientPackage.java:63]: Remove

File: src/main/resources/static/scripts/ajaxFunctions.js [hunk 7]
```diff
@@ -144,7 +144,7 @@ function useClientPackageElement(row){
 }
 
 function ajaxAddPackageToClient(packageId, packagePrice){
-	import { COLOR_GREEN, COLOR_YELLOW } from 'constants';
+//	import { COLOR_GREEN, COLOR_YELLOW } from 'constants';
```
Comment (renatav) [inline src/main/resources/static/scripts/ajaxFunctions.js:147]: why was this removed?

Comment (PerAsperaAdAstra3) [reply src/main/resources/static/scripts/ajaxFunctions.js:147]: In this form it does not work, while it was active the page could not be generated.
